src/content/platform-integration/platform-channels.md

```diff
@@ -894,7 +894,9 @@ And replace with the following:
 You should now be able to run the application on Windows.
 If your device doesn't have a battery,
 it displays 'Battery level not available'.
-  
+Note: To ensure the changes take effect, the project should be recompiled 
+with Visual Studio.Otherwise, you may encounter an error saying: Failed to get 
+battery level: Memory leak .
```

Comment:
```suggestion
:::note
To ensure that the changes take effect, recompile the project.
Otherwise, you might encounter the following error: **Failed to get 
battery level: Memory leak.**
:::
```